packages/next/src/client/components/request-async-storage.external.ts

```diff
@@ -48,6 +50,11 @@ export { requestAsyncStorage }
 export function getExpectedRequestStore(callingExpression: string) {
   const store = requestAsyncStorage.getStore()
   if (store) return store
+  if (cacheAsyncStorage.getStore()) {
+    throw new Error(
+      `\`${callingExpression}\` cannot be called inside "use cache". Call it outside and pass an argument instead. Read more: https://nextjs.org/docs/messages/next-request-in-use-cache`
```

Comment:
Are we creating this page in this PR or separately? I believe the files in this directory map to `docs/messages`:
https://github.com/vercel/next.js/tree/canary/errors

Reply:
I added an error page.